src/ontology/uberon-odk.yaml

```diff
@@ -44,7 +44,7 @@ import_group:
     - id: cl
       use_base: TRUE
     - id: go
-      use_base: TRUE
+      make_base: TRUE
```

Comment:
```suggestion
      make_base: TRUE
      mirror_from: http://purl.obolibrary.org/obo/go/go-base.owl
```

Reply:
unfortunately, you have not been properly introduced to the GO ontology structure. What is sold on http://purl.obolibrary.org/obo/go.owl is a drastically reduced version of GO which is lacking a lot of the rich links to other ontologies. So you need to add _mirror_from_ as well here. http://purl.obolibrary.org/obo/go/go-base.owl has _more_, not _less_ then http://purl.obolibrary.org/obo/go.owl.. Gr.. Sorry. 

In any case, hopefully, you will still get _some_reduction, just not as much as you are hoping :)

Reply:
but we don't need the link to other ontologies when importing GO, right?

Reply:
They drive inference! So those axioms will contribute to reasoning of terms involving GO terms (especially taxon stuff, but also many other things). So yes, conceptually at least we need them. Practically, no one knows.

Reply:
yeah, I didn't see any difference for Uberon when using the `go.owl`, but that's fine.